extensions/amp-date-picker/0.1/dates-list.js

```diff
@@ -14,16 +14,20 @@
  * limitations under the License.
  */
 
+import * as rrule from '../../../node_modules/rrule/dist/es5/rrule.min.js';
 import {requireExternal} from '../../../src/module';
 
+const rrulestr = rrule.default.rrulestr || rrule.rrulestr; // CC imports into .default, browserify flattens a layer.
```

Comment:
Nit: Please move this into `tryParseRrulestr`.

Reply:
I think it makes more sense where it is, considering ideally it would even be `import {rrulestr} from 'rrule';`